src/sidebar/components/sidebar-content.js

```diff
@@ -74,6 +76,11 @@ function SidebarContent({
     hasDirectLinkedAnnotationError || hasDirectLinkedGroupError;
 
   const showTabs = !hasContentError && !hasAppliedFilter;
+  const showFilterStatus = features.flagEnabled('client_filter_status');
```

Comment:
I would recommend using `store.isFeatureEnabled` in new code rather than the `features` service, as otherwise the component won't be re-rendered if the feature flag changes.